README.md

```diff
@@ -64,19 +63,21 @@ export default {
     [
       "onesignal-expo-plugin",
       {
-        mode: process.env.NODE_ENV || "development",
-        devTeam: "91SW8A37CR"
+        mode: "development",
       }
     ]
   ]
 };
 ```
 
-#### Plugin Options
-* `mode`: used to configure [APNs environment](https://developer.apple.com/documentation/bundleresources/entitlements/aps-environment) entitlement.
-   - `"development"`
-   - `"production"`
-* `devTeam`: *optional* - used to configure Apple Team ID. You can find your Apple Team ID by running `expo credentials:manager`.
+#### Plugin Prop
+You can pass props to the plugin config object to configure:
+
+| Plugin Prop              |          |                                                                                                                                                                                                                                                                                                                                |
+|--------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `mode`                   | **required** | Used to configure  [APNs environment](https://developer.apple.com/documentation/bundleresources/entitlements/aps-environment)  entitlement.  `"development"` or  `"production"`                                                                                                                                                |
+| `devTeam`                | optional | Used to configure Apple Team ID. You can find your Apple Team ID by running `expo credentials:manager`  e.g: `"91SW8A37CR"`                                                                                                                                                                                                    |
+| `iPhoneDeploymentTarget` | optional | Target `IPHONEOS_DEPLOYMENT_TARGET` value to be used when adding the iOS [NSE](https://documentation.onesignal.com/docs/service-extensions). A deployment target is nothing more than the minimum version of the operating system the application can run on. This value should match the value in your Podfile e.g: `"12.0"`. |
```

Comment:
> This value should match the value in your Podfile e.g: `"12.0"`. |

Is there another place an Expo developer would find this? Those who use EAS won't have access to the `Podfile` to find this value.

Reply:
Ideally, we would find a table that maps the Expo version with the version used in the Podfile. We may have to do some hunting for this.